actions/threads.tsx

```diff
@@ -125,11 +122,6 @@ export async function createThread(
   );
   await fs.writeFile(path.join(threadPath, STATE_FILE), '');
 
-  if (firstMessage) {
-    const generatedThreadName = await generateThreadName(firstMessage);
```

Comment:
As I understand it, it was a requirement to have the LLM name the thread based on the first message. Has that requirement changed?

Reply:
Hmm, I changed it because the thread will always be created at first and it is kind of weird that the name will be changed later after user type the message. But yeah I will confirm with craig.

Reply:
will change this, thanks for pointting out